#########################
# SDI state message to HDI
#########################
float64 wheel_angle	# [-5*PI,+5*PI] in radians
float64 wheel_force	# [-20.0, 20.0] Nm
float64 vibration	# [0.0, 1.0]
